src/components/ContentPreview/ContentPreview.js

```diff
@@ -29,6 +29,8 @@ import { getTypedFileId } from '../../util/file';
 import { withErrorBoundary } from '../ErrorBoundary';
 import ReloadNotification from './ReloadNotification';
 import { PREVIEW_FIELDS_TO_FETCH } from '../../util/fields';
+import { FeatureProvider } from '../FeatureChecking';
+import type { FeatureConfig } from '../FeatureChecking';
```

Comment:
Can you move this import to `flow-typed/box-ui-elements.js` and see if flow can detect it implicitly?

Reply:
Oh, we consume our own flow-typed definition? That's normally for third-party code: https://flow.org/en/docs/libdefs/creation/#declaring-a-global-function-

Reply:
That's the convention today, at least. Seems like something to follow for now and refactor holistically in a separate PR.